src/libs/actions/Session/index.js

```diff
@@ -23,6 +23,32 @@ Onyx.connect({
     callback: val => credentials = val || {},
 });
 
+/**
+ * Manage push notification subscriptions on sign-in/sign-out.
+ *
+ * We would normally do this when AuthScreens mounts and unmounts. However on Android,
+ * AuthScreens unmounts when the app is closed with the back button so we manage the
+ * push subscription when the session changes here.
+ */
+let previousAccountID = '';
```

Comment:
I think accountID are number?

Reply:
Maybe can make it `undefined` initially? 

```suggestion
let previousAccountID;
```

Reply:
> I think accountID are number?

Yeah that's actually right it looked like a string in logs